packages/kolme/src/core/kolme.rs

```diff
@@ -1186,6 +1188,22 @@ impl<App: KolmeApp> Kolme<App> {
         }
         Ok(next)
     }
+
+    async fn init_remote_data_listener(&self) -> Result<()> {
+        self.inner.store.init_remote_data_listener().await?;
+        let mut sub = self.inner.store.subscribe_remote_data();
+        let kolme = self.clone();
+        tokio::spawn(async move {
```

Comment:
Only concern here is that this is an infinite background task that can never be stopped. In practice, I can't think of a time when that would be an issue. Using some kind of a weak reference in here would be a "fix" for that problem if we wanted to address it.